community/kernel/src/main/java/org/neo4j/kernel/impl/api/index/MultipleIndexPopulator.java

```diff
@@ -311,18 +317,17 @@ public void flipAfterPopulation()
 
     private int[] propertyKeyIds()
     {
-        return populations.stream().flatMapToInt( this::propertyKeyIds ).distinct().toArray();
+        return populations.keySet().stream().flatMapToInt( this::propertyKeyIds ).distinct().toArray();
     }
```

Comment:
This method is actually pretty inefficient, it is only ever used together with `labelIds()`, it would be better to just have a single for-loop build both arrays at once, the streams just add overhead.